glusterd2/xlator/validation.go

```diff
@@ -52,6 +52,29 @@ func validateBitrot(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateQuota(v *volume.Volinfo, key string, value string) error {
+
+	// Check if quota is already enabled
+	if volume.IsQuotaEnabled(v) == false {
+		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
+		return err
+	}
+
+	switch key {
+	case "deem-statfs":
+		return nil
+	case "soft-timeout":
+		return nil
+	case "hard-timeout":
+		return nil
+	case "alert-time":
+		return nil
+	case "default-soft-limit":
+		return nil
+	}
```

Comment:
There should be a `case default` which returns error OR the last return outside the switch block must return error.

Reply:
done